e2e/test/prerequisites/readme.md

```diff
@@ -1,11 +1,34 @@
+
 # Azure IoT C# End-to-end test prerequisites
```

Comment:
This file is completely changed. So a good way to look at it is - View file option to read through the contents and add feedback.